src/Microsoft.AspNetCore.Mvc.Core/Authorization/AuthorizeFilter.cs

```diff
@@ -113,6 +113,20 @@ public AuthorizeFilter(string policy)
             }
 
             var effectivePolicy = Policy;
+            if (context.CombineAuthorizeFilters)
```

Comment:
This should be part of MvcOptions and global for the app, either you are aware of this behavior and want it, or things should work as is.

Reply:
If all of the new behavior is in here, then this class can just read options

Reply:
Ok that's easier, so just service locate the MvcOptions from HttpContext and then I can get rid of plumbing the options everywhere?

Reply:
Yes, that would be better